cmd/kube-controllers/main.go

```diff
@@ -121,8 +130,64 @@ func main() {
 	// If configured to do so, start an etcdv3 compaction.
 	startCompactor(ctx, config)
 
-	// Wait forever.
-	select {}
+	sigint := make(chan os.Signal, 1)
+	signal.Notify(sigint, os.Interrupt)
+
+	// Wait forever and perform healthchecks.
```

Comment:
So, curious - is this construct giving us any benefits relative to a simple sleep? We're basically replacing a sleep.

Would this work?

```
for {
  <do some stuff>
  time.Sleep(10 * time.Second)
}
```

The benefit there is that we don't wait 10 seconds before performing our first health check. WDYT?